ModelBundle/Validator/Constraints/UniqueNodeOrderValidator.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\Validator\Constraints;
+
+use OpenOrchestra\ModelInterface\Model\NodeInterface;
+use OpenOrchestra\ModelInterface\Repository\NodeRepositoryInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+/**
+ * Class UniqueNodeOrderValidator
+ */
+class UniqueNodeOrderValidator extends ConstraintValidator
```

Comment:
As your constraint is not linked to any mongo logic, I would rather put it insite the `BackofficeBundle`, this way, it will still be active if you modify the database